sql/qaqc/qaqc_field_distribution.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS field_dist_qaqc;
+CREATE TABLE IF NOT EXISTS field_dist_qaqc(
+    field_name character varying,
+    result character varying
+);
+
+INSERT INTO field_dist_qaqc (
+    SELECT 'Job_Type' as field_name,
+            jsonb_agg(json_build_object('job_type',tmp.job_type,'count',tmp.count)) as result
+    FROM(
+        SELECT job_type, COUNT(DISTINCT job_number) as count
+        FROM final_devdb
+        WHERE date_lastupdt >= CAST(:PRE_DATE AS VARCHAR)
+        GROUP BY job_type) tmp );
+
+INSERT INTO field_dist_qaqc (
+    SELECT 'Job_Status' as field_name,
+            jsonb_agg(json_build_object('job_status',tmp.job_status,'count',tmp.count)) as result 
+    FROM(
+        SELECT job_status, COUNT(DISTINCT job_number) as count
+        FROM final_devdb
+        WHERE date_lastupdt >= CAST(:PRE_DATE AS VARCHAR)
```

Comment:
Ran the code on my machine and the schema of the output looks good, but don't we want to cast `date_lastupdt` to date instead of casting previous date to string?

Reply:
We can cast it to date. But casting to string is necessary to do the date comparison. The pass in variable seems to be integer type and a timezone needs to be specified in order to cast it to timestamp.

Reply:
Sorry I don't understand. Don't we want date type to do the date comparison? We don't need to worry about timezones for a timestamp

Reply:
I think as long as they are of same format 'YY-MM-DD', we can use two strings to do the comparison if we want to code simpler. Let me research on this a little bit.